test/test-applications/integrations/dependency-libs/LogsInjectionHelper/LoggingMethods.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.IO;
+using System.Reflection;
+using Datadog.Trace;
+using Datadog.Trace.Configuration;
+
+namespace PluginApplication
+{
+    public static class LoggingMethods
+    {
+        /// <summary>
+        /// Prepend a string to log lines that should not be validated for logs injection.
+        /// In other words, they're not written within a Datadog scope 
+        /// </summary>
+        private static readonly string ExcludeMessagePrefix = "[ExcludeMessage]";
+
+        public static void DeleteExistingLogs()
+        {
+            var appDirectory = Directory.GetParent(typeof(LoggingMethods).Assembly.Location).FullName;
+            var textFilePath = Path.Combine(appDirectory, "log-textFile.log");
+            var jsonFilePath = Path.Combine(appDirectory, "log-jsonFile.log");
+
+            File.Delete(textFilePath);
+            File.Delete(jsonFilePath);
+        }
+
+        public static int RunLoggingProcedure(Action<string> logAction, bool makeCrossAppDomainCall = true)
+        {
+#if NETFRAMEWORK
+            // Set up the secondary AppDomain first
+            // The plugin application we'll call was built and copied to the ApplicationFiles subdirectory
+            // Create an AppDomain with that directory as the appBasePath
+            var entryDirectory = Directory.GetParent(Assembly.GetEntryAssembly().Location);
+            var applicationFilesDirectory = Path.Combine(entryDirectory.FullName, "ApplicationFiles");
+            var applicationAppDomain = AppDomain.CreateDomain("ApplicationAppDomain", null, applicationFilesDirectory, applicationFilesDirectory, false);
+#endif
+
+            // Set up Tracer and start a trace
+            var settings = TracerSettings.FromDefaultSources();
+            settings.LogsInjectionEnabled = true;
+            settings.Environment ??= "dev"; // Later we can test when Environment=null / dd.env=null
```

Comment:
FYI, by default, when starting sample apps, we set `DD_ENV` to `integration_tests` for consistency: 
https://github.com/DataDog/dd-trace-dotnet/blob/master/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs#L217

Reply:
Yep 👍🏼 I will add a comment to remove any confusion. I wrote this to make sure local dev testing had it assigned.

Fixed in 58967e1c9556e1eda67a978f2acb5a2a7713381f